src/get-started/create-service.md

```diff
@@ -86,42 +93,114 @@ const INIT_BALANCE: u64 = 100;
 
 ## Declare Persistent Data
 
+Exonum uses Protobuf as its [serialization format](../architecture/serialization.md)
+for storage of data. Thus, we need to describe our structures using the Protobuf
+interface description language first. The corresponding Rust structures will be
+later generated from them.
+
 We should declare what kind of data the service will store in the blockchain.
 In our case we need to declare a single type – *wallet*.
 Inside the wallet we want to store:
 
-- **Public key** to validate requests from the owner of the wallet
+- **Public key** which is the address of the wallet
 - **Name of the owner** (purely for convenience reasons)
-- **Current balance** of the wallet
+- **Current balance** of the wallet.
+
+As a first step we add a module named `proto` to our project. We add a
+`cryptocurrency.proto` file to this module and describe the `Wallet` structure
+in it in the Protobuf format. The `Wallet` datatype will look as follows:
+
+```protobuf
+syntax = "proto3";
 
-Summing it all up, the `Wallet` datatype will look like:
+// Allows to use `exonum.PublicKey` structure already described in `exonum`
+// library.
+import "helpers.proto";
+
+// Wallet structure used to persist data within the service.
+message Wallet {
+  exonum.PublicKey pub_key = 1;
+  string name = 2;
+  uint64 balance = 3;
+}
+```
+
+Secondly, to generate a Rust structure from the above-stated definition, we add
+a `mod.rs` file with the following content to the `proto` module:
 
 ```rust
-encoding_struct! {
-    struct Wallet {
-        pub_key: &PublicKey,
-        name: &str,
-        balance: u64,
-    }
+#![allow(bare_trait_objects)]
+#![allow(renamed_and_removed_lints)]
+
+pub use self::cryptocurrency::Wallet;
```

Comment:
Maybe It makes sense to use all structs from the beginning, including: TxCreateWallet, TxTransfer?